api/engineapi/cli.py

```diff
@@ -380,6 +382,58 @@ def create_leaderboard_handler(args: argparse.Namespace) -> None:
 #         print(f"Amount of updated claimants: {len(claimant_signature)}")
 
 
+def assign_resource_handler(args: argparse.Namespace) -> None:
+
+    with db.yield_db_session_ctx() as db_session:
+
+        try:
+            resource_id = actions.assign_resource(
+                db_session=db_session,
+                resource_id=args.resource_id,
+                leaderboard_id=args.leaderboard_id,
+            )
+            logger.info(
+                f"leaderboard:{args.leaderboard_id} assign resource_id:{resource_id}"
+            )
+        except Exception as err:
+            logger.error(f"Unhandled /assign_resource exception: {err}")
+            return
+
+
+def list_resources_handler(args: argparse.Namespace) -> None:
+
+    with db.yield_db_session_ctx() as db_session:
+        resources = actions.list_leaderboards_resources(db_session=db_session)
+
+        print(resources)
```

Comment:
In function above you are logging, here printing, better to use one common approach.